uaclient/tests/test_cli_status.py

```diff
@@ -965,6 +978,30 @@ def contract_info_side_effect(cfg, token):
 
         assert output["errors"][0]["message"] == warning_message
 
+    @pytest.mark.parametrize("contract_changed", (False, True))
```

Comment:
Let's also add unittest for the `is_contract_changed` function directly. Just to see if it covers the two scenarios we care about: date and entitlements changes

Reply:
@CalvoM Thanks for the additional tests, but I think we are still missing the `contract.is_contract_changed` tests